packages/instantsearch-ui-components/src/components/__tests__/FrequentlyBoughtTogether.test.tsx

```diff
@@ -62,16 +62,20 @@ describe('FrequentlyBoughtTogether', () => {
               <li
                 class="ais-FrequentlyBoughtTogether-item"
               >
-                <div>
-                  1
-                </div>
+                <li>
+                  <div>
+                    1
+                  </div>
+                </li>
```

Comment:
The `ItemComponent` provided in the test could be updated to not wrap the data within a `<li>`.